### PR TITLE
Force quit frontend server after one second, unskip tests

### PR DESCRIPTION
**Pull request by jakemac53**
Fixes https://github.com/dart-lang/webdev/issues/1386
Fixes https://github.com/dart-lang/webdev/issues/1383

### CODE REVIEW

**Comment by jakemac53** [comment]
Looks like I still see at least one timeout here. Probably this needs some investigation on a windows box.

**Comment by jakemac53** [comment]
I tried running the test that timed out locally and the whole thing completed in 2.5 seconds. So I do not believe it is timing out it is actually hanging. Trying to figure out why, but I can't reproduce it locally.

**Comment by jakemac53** [comment]
Ahah! It appears to actually be hanging on the cleanup of the client (ie: in the `tearDown` code).

File: frontend_server_client/lib/src/frontend_server_client.dart [hunk 1]
```diff
@@ -295,10 +295,13 @@ class FrontendServerClient {
   }
 
   /// Stop the service gracefully (using the shutdown command)
-  Future<int> shutdown() {
-    _feServerStdoutLines.cancel();
+  Future<int> shutdown() async {
```
Comment (annagrin) [inline frontend_server_client/lib/src/frontend_server_client.dart:298]: It would be great to open a bug on the SDK for that!

**Review by annagrin — APPROVED** [review]
LGTM with a comment

**Review by natebosch — APPROVED** [review]

**Pull request merged**